### PR TITLE
Adding mac support

### PR DESCRIPTION
**Pull request by AnthonyLim23**
This PR adds support for mac. I have cleaned up some code, added the steps needed for creating mac packages and updated the Readme. 

To review, look at the code and see that https://github.com/mantidproject/quickBayes/pull/58/commits/753330b136b997134b8b6801a11f5291049c5199 created the release in https://pypi.org/project/quickBayes/#history

Conor has also tested doing a pip install of 1.0.0b5 on a mac and it seems to work.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,10 +12,11 @@
 setup(
     name=PACKAGE_NAME,
     requires=['numpy'],
+    setup_requires=['numpy>=1.12'],
     install_requires=['numpy>=1.12', 'scipy', 'gofit'],
     packages=find_packages(where='src'),
     description='A Bayesian fitting package used for '
-                'model selection and grid searches of fits '
+                'model selection and grid searchs of fits '
```
Comment (AnthonyLim23) [inline setup.py:19]:
```suggestion
                'model selection and grid searches of fits '
```

File: README.md [hunk 2]
```diff
@@ -51,16 +55,11 @@ You should now be able to run the tests:
 > python -m pytest .
 ```
 
-
 ## Building for PyPi
 
-If this is your first time interacting with PyPi then please see [here](https://packaging.python.org/en/latest/tutorials/packaging-projects/#uploading-the-distribution-archives) for instructions of how to setup accounts and API tokens. The package will automatically be uploaded when a relase is created in github.
+If this is your first time interacting with PyPi then please see [here](https://packaging.python.org/en/latest/tutorials/packaging-projects/#uploading-the-distribution-archives) for instructions of how to setup accounts and API tokens.
+The package will automatically be uploaded when a relase is created in github, see [here](https://cibuildwheel.readthedocs.io/en/stable/options/) for the details.
```
Comment (robertapplin) [inline README.md:61]:
```suggestion
The package will automatically be uploaded when a release is created in Github, see [here](https://cibuildwheel.readthedocs.io/en/stable/options/) for the details.
```

File: README.md [hunk 3]
```diff
@@ -23,22 +23,26 @@ In your terminal, but outside of your environment, type `` mamba env update --fi
...
 In your terminal (Gitbash on Windows) you will now be able to run pre-commit checks (outside of your Conda environment).
 
-### Linux
+### Setup
 
-Create a minimal conda environment named `quickBayes-dev` (you can use another name if you prefer):
+
+This project uses a minimal conda environment for development called `quickBayes-dev` (you can use another name if you prefer):
+
+To create the environment run the command from the root directory of the project:
```
Comment (robertapplin) [inline README.md:31]:
```suggestion
To create the environment run this command from the root directory of the project:
```

**Review by robertapplin — CHANGES_REQUESTED** [review]
Looks good. Just spotted a couple of typos

**Review by robertapplin — APPROVED** [review]
Thanks for the changes, looks good to me

**Pull request merged**